core/store/src/db.rs

```diff
@@ -692,8 +692,8 @@ impl RocksDB {
     }
 
     /// Returns version of the database state on disk.
-    pub fn get_version(path: &Path) -> io::Result<DbVersion> {
-        let value = RocksDB::open(path, &StoreConfig::default(), Mode::ReadOnly)?
+    pub fn get_version(path: &Path, config: &StoreConfig) -> io::Result<DbVersion> {
```

Comment:
While at it, could you make it `pub(crate)`?

Reply:
done.